index.js

```diff
@@ -4,66 +4,74 @@ const os = require('os');
 const fs = require('fs');
 const ini = require('ini');
 
+const isWindows = process.platform === 'win32';
```

Comment:
In Line: 111:
```js
exports.npm.binaries = process.platform === 'win32' ? npmPrefix : path.join(npmPrefix, 'bin');
```
Replace `process.platform === 'win32'` to `isWindows`

Reply:
Done 👍